ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -747,3 +747,8 @@ def remainder(self, y, name=None):
 
     def is_floating_point(self):
         return paddle_frontend.is_floating_point(self._ivy_array)
+
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
```

Comment:
Hey @samthakur587 . There's a conflict here. It's a simple one. Can you resolve it please! 😄

Reply:
Hi! @HaiderSultanArc i resolved the conflict. now its ready to merge. 🚀

Reply:
Hey @samthakur587 your tests are failing now. Can you have a look? Only look for your failing tests. You can ignore others. Thanks 😄

Reply:
Hi! the failing test is not related to my PR. what's happen i thing someone has made changes to testing pipeline. in local its showing this error.

![image](https://github.com/unifyai/ivy/assets/83540902/84848943-3869-4137-a84c-7678e595186b)

I don't know what happening but its start after i pull the latest changes from ivy. 😄

Reply:
@sherry30 🤔 do you know something about this?